features-json/maxlength.json

```diff
@@ -0,0 +1,240 @@
+{
+  "title":"maxlength attribute for input and textarea elements",
+  "description":"Declares an upper bound on the number of characters the user can input. Normally the UI ignores attempts by the user to type in additional characters beyond this limit.",
+  "spec":"https://html.spec.whatwg.org/multipage/forms.html#attr-input-maxlength",
+  "status":"ls",
+  "links":[
+    {
+      "url":"https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-maxlength",
+      "title":"Mozilla Developer Network"
+    }
+  ],
+  "bugs":[
+  ],
+  "categories":[
+    "HTML5",
+    "JS API",
```

Comment:
`maxlength` is an attribute, `.maxLength` is a DOM property, `ValidityState.tooLong` feels like general API. That was my thinking anyway.